app/controllers/application_controller.rb

```diff
@@ -655,6 +656,18 @@ def check_xhr
     raise ApplicationController::RenderEmpty.new unless ((request.format && request.format.json?) || request.xhr?)
   end
 
+  def block_cdn_requests
+    raise Discourse::NotFound if Discourse.is_cdn_request?(request.env)
+  end
+
+  def add_cors_header
+    if Discourse.is_cdn_request?(request.env) && ["GET", "OPTIONS"].include?(request.method)
+      response.headers['Access-Control-Allow-Origin'] = '*'
+      response.headers['Access-Control-Allow-Credentials'] = 'false'
```

Comment:
I don't think we need this: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Credentials

> If you don't need credentials, omit this header entirely (rather than setting its value to false).